Automattic/pocket-casts-ios
# Onboarding: Includes some small improvements

| 📘 Project: #500 |
|:---:|

- Fixes 2 step 5's for the Apple Shortcuts steps
- Moves the terms link to a new line on the purchase options
- Saves the newsletter status when the user swipes to dismiss the welcome view

## To test

1. Enable the `tracksLoggingEnabled` feature flag
2. Launch the app and logout
3. Add `Settings.shouldShowInitialOnboardingFlow = true`
4. Launch the app again
5. Create a new account
6. Go through the flow and tap Unlock on the plus view
7. ✅ Verify you see the terms on a new line
8. Complete the purchase
9. Go to the Welcome view
10. Tap on Import
11. Tap on Apple Shortcuts
12. ✅ Verify the text does not have 2 step 5's
13. Go back
14. Swipe down to dismiss the welcome view
15. ✅ Verify you see ` 🔵 Tracked: newsletter_opt_in_changed ["enabled": true, "source": "welcome_plus"]`

## Checklist
